android/src/main/java/com/optimove/flutter/OptimoveInitProvider.java

```diff
@@ -0,0 +1,244 @@
+package com.optimove.flutter;
+
+import android.app.Application;
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.content.res.AssetManager;
+import android.database.Cursor;
+import android.net.Uri;
+import android.util.JsonReader;
+import android.util.JsonToken;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.optimove.android.Optimove;
+import com.optimove.android.OptimoveConfig;
+import com.optimove.android.optimobile.DeferredDeepLinkHandlerInterface;
+import com.optimove.android.optimobile.OptimoveInApp;
+
+import org.json.JSONException;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.optimove.flutter.OptimoveFlutterSdkPlugin.eventSink;
+
+public class OptimoveInitProvider extends ContentProvider {
+    private static final String TAG = OptimoveInitProvider.class.getName();
+    private static final String OPTIMOVE_CREDENTIALS = "optimoveCredentials";
+    private static final String OPTIMOBILE_CREDENTIALS = "optimobileCredentials";
+    private static final String KEY_IN_APP_CONSENT_STRATEGY = "inAppConsentStrategy";
```

Comment:
Consistency, prefix all keys w/ `KEY_` ?

Reply:
Changed all to XX_KEY